### PR TITLE
Conditionally add Activity Log card to dashboard if is an active feature

### PR DESCRIPTION
**Pull request by dereksmart**
This checks the `/features` endpoint response to see if `rewind` is active for the given site.  If so, it will add a card to the dashboard, the details of which are explained in #8199

more testing instructions when it's ready.

### CODE REVIEW

**Comment by eliorivero** [comment]
This should be updated to use this new constant
https://github.com/Automattic/dops-components/pull/118

File: _inc/client/at-a-glance/index.jsx [hunk 1]
```diff
@@ -53,6 +56,7 @@ class AtAGlance extends Component {
 			siteAdminUrl: this.props.siteAdminUrl,
 			siteRawUrl: this.props.siteRawUrl
 		};
+		const isRewindActive = includes( this.props.activeFeatures, 'jetpack-rewind' );
```
Comment (eliorivero) [inline _inc/client/at-a-glance/index.jsx]: This should be updated to use this new constant
https://github.com/Automattic/dops-components/pull/118

Comment (oskosk) [reply _inc/client/at-a-glance/index.jsx]: Addressed by the changes in b5b3ab2e7a4d11004157868e16b73cb31efb0881 for now

Comment (oskosk) [reply _inc/client/at-a-glance/index.jsx]: Just for the records, the feature is still called `jetpack-rewind` so b5b3ab2 was removed

**Comment by oskosk** [comment]
Cherry-picked to `branch-5.6` in 69335b88d714745d853eb4c444a7aa875449f4de

**Pull request merged**